src/blocks/events/block.json

```diff
@@ -22,6 +22,10 @@
 		},
 		"customTextColor": {
 			"type": "string"
+		},
+		"showCarousel": {
+			"type": "boolean",
+			"default": true
```

Comment:
I don't think that we need to have this as an attribute because we can base it on the `externalCalendarUrl` attribute whether or not to show the carousel. By not using attributes wherever possible we avoid having to deal with deprecations in the future related to the attribute.

Reply:
I see now what you are doing with this attribute. Sounds reasonable that we should have this as an attribute given its use on the front-end. Concerning this feature however, I think that we should aim to achieve the same aesthetic output for the Events block in the editor as we have in the front-end. That being said we should not render what looks like a carousel in the editor, regardless of edit state, when the attribute `showCarousel` is false.